File: Códigos/Ouve Fácil-SRC_01/class/subCategoria.php
<?php

class subCategoria{
    private $codSubCategoria;
    private $nome;

    public function setCodSubCategoria($codSubCategoria){
        $this->codSubCategoria = $codSubCategoria;
    }
    public function getCodSubCategoria(){
        return $this->codSubCategoria;
    }
    public function setNome($nome){
        $this->nome = $nome;
    }
    public function getNome(){
        return $this->nome;
    }
}
?>

File: Códigos/OuveFacil/OuveFacil/app/src/main/java/domain/model/SubCategoria.java
package domain.model;

public class SubCategoria {

    private int codSubCategoria;
    private String nome;
    private Categoria categoria;

    public int getCodSubCategoria() {
        return codSubCategoria;
    }

    public void setCodSubCategoria(int codSubCategoria) {
        this.codSubCategoria = codSubCategoria;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Categoria getCategoria() {
        return categoria;
    }

    public void setCategoria(Categoria categoria) {
        this.categoria = categoria;
    }

    @Override
    public String toString() {
        return this.nome;
    }
}
